tools/parse/src/main/scala/org/allenai/nlpstack/parse/poly/decisiontree/RandomForest.scala

```diff
@@ -100,7 +101,13 @@ class RandomForestTrainer(validationPercentage: Double, numDecisionTrees: Int,
     * @return the induced random forest
     */
   override def apply(data: FeatureVectorSource): ProbabilisticClassifier = {
-    val subtreeFiles = Seq.fill(numDecisionTrees) {
+    import scala.concurrent.duration._
+    import scala.concurrent._
+    import scala.language.postfixOps
+    implicit val executionContext = ExecutionContext.fromExecutor(
```

Comment:
Question: how do I specify the number of threads in the default execution context?